demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -172,6 +172,27 @@ def validate_content_type_header(self, content_type: str) -> bool:
                 return False
         return True
 
+    @error_codes("RN116")
+    def validate_first_level_header_exists(self) -> bool:
+        """
+            Validate that the RN has a first level header.
+        Return:
+            True if the RN has a first level header, False otherwise.
+        """
+        first_level_header_index = re.search(
+            r"\s#{4,4}\s", f"\n{self.latest_release_notes}"
```

Comment:
can we replace `\s` with `\b`?

Reply:
Switched to \B since this is a non-word boundary.